__tests__/components/ui/SideNavigation/_partials/Content.test.tsx

```diff
@@ -0,0 +1,34 @@
+import { NavItem } from "@/components/ui/SideNavigation/_partials/Content";
+import "@testing-library/jest-dom";
+import { render, screen } from "@testing-library/react";
+
+describe("Content", () => {
+  const item = {
+    label: "Dummy",
+    subitems: [{ label: "Dummy", link: "/dummy" }],
+  };
+  it("should render content", () => {
+    render(<NavItem item={item} />);
+    const content = screen.getByTestId("contentId");
+    expect(content).toBeInTheDocument();
+  });
+
+  it("should render style when active", () => {
+    render(<NavItem item={item} isActive />);
+    const content = screen.getByTestId("contentId");
+    expect(content.children[0]).toHaveAttribute("style", "display: !inline-block;");
+  });
```

Comment:
This is hard to follow, let's avoid testing styles, the active style can change at anytime while the component still have the same behavour